coap/examples/coap_client/main/certs/coap_client.key

```diff
@@ -1,4 +1,5 @@
 -----BEGIN PRIVATE KEY-----
```

Comment:
Any specific reason to update the private keys in the examples? I thought updating the server cert should fix the connection problem. @mrdeep1

Reply:
> A thing to note here is that the client and server are set up to mutually trust each other here. The client checks the server certificates and the server checks the client certificates AND that they are signed by the same CA before proceeding.

Aha, got it. I somehow missed on this point!